### ISSUE TITLE
IOS Quick Paste

### ISSUE THREAD

**Issue by mohdtaha60** [iamvivekkaushik/OTPTextField#7]
When we clicking on Quick Paste to OTP from IOS Keyboard, It's only pasting the First digit. Can you please fix this.
![Screenshot 2020-11-05 143258](https://user-images.githubusercontent.com/49991948/98220085-d3cc1700-1f73-11eb-96dc-d386f610a8a9.png)

**Comment by iamvivekkaushik** [comment]
Thanks for bringing this to my attention. I'll fix this once I get some free time on hand.

**Comment by mohdtaha60** [comment]
Thanks, I hope you do this asap.

**Comment by hims1008** [comment]
is any solution found for quick paste?

**Comment by mohdtaha60** [comment]
Not yet! Seems developer isn't getting anytime free on hand

**Comment by iamvivekkaushik** [comment]
Yeah, sorry guys. I've implemented a quick fix that still contains some bugs that need to be fixed. You can use the latest Github version of this library in your project.

**Comment by lets-swapcode** [comment]
any update on this ? @iamvivekkaushik bro ?

**Comment by iamvivekkaushik** [comment]
> any update on this ? @iamvivekkaushik bro ? 

It works currently, there are just few bugs (edge cases), that I haven't fixed yet. You can use the latest version of the package.

**Comment by rohitmagic10** [comment]
FYI, it doesn't work correctly. It picks only 1 digit from the keyboard one time code.

**Comment by jacekendrick** [comment]
Still having this issue. I'm on the current package version (1.1.3)

**Comment by lovepreetSinghElectricpe** [comment]
Try removing ** maxLength: 1,** on line 189 from otp_field.dart file, and it works fine.

**Comment by LobnaShaheen** [comment]
@lovepreetSinghElectricpe 

This solution did not work for me any new ideas for this issue.

**Comment by Lulzphantom** [comment]
Any update on this issue?
